install/install.py

```diff
@@ -26,6 +26,22 @@ def __init__(self, RootDir):
         self.parser.add_argument('--no-user-input', help='run script with default options for user input', action="store_true")
         self.parser.add_argument('--core-only', help='install only owtf dependencies, skip optional tools', action="store_true")
 
+    def print_success(self, error_code):
```

Comment:
function name should be intuitive. `print_success` tells that the installation was successful. Yet you have error handling in the this.

Better rename it to be: `print_finish` or something like that.